packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -943,13 +945,16 @@ class AnimatedPhysicalModel extends ImplicitlyAnimatedWidget {
     @required this.elevation,
     @required this.color,
     this.animateColor: true,
+    @required this.shadowColor,
+    this.animateShadowColor: true,
```

Comment:
We're not asserting for not null for `animateShadowColor` because `animateColor` doesn't have one, but maybe they should both be asserting?

Reply:
Good point. Let's assert. `true` or `false` is all we need for these two.

Reply:
dc9759f0ead8e18a50544fea4f5edb8f5e2d4248